Dockerfile-gnome

```diff
@@ -0,0 +1,69 @@
+FROM ruby:2.5.1
```

Comment:
I'm not sure, but I think the Dockerfile just has to be named Dockerfile.

Reply:
Never mind, I see they have a Dockerfile-alpine as well.

Reply:
You can run it with `docker build -f 'name'`, it's meant to preserve upstream's build environment.